omod/src/main/java/org/openmrs/sync/rest/CreateChildServerRestController.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.sync.rest;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.sync.api.SyncService;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+@RestController
+public class CreateChildServerRestController {
```

Comment:
Looks good @cioan .  Maybe rename this class to "DownloadDbRestController" or "GenerateDataFileRestController"?  This doesn't create a child server.

Reply:
This is also a very dangerous endpoint.  We need to ensure it is tightly privileged.

Reply:
I just added 'Manage Synchronization' as a required privilege. Let me know if I should use a different privilege? 
I also refactor the controller name.